### PR TITLE
Add Codebundles for handling Laggy Patroni Replicas

### PR DESCRIPTION
**Pull request by jon-funk**
- Adds an SLI for fetching the maximum lag (in MB) among replicas in a Patroni cluster within a Kubernetes cluster
- Adds a taskset which can automatically remediate laggy replicas by reinitializating them using `patronictl` from a Kubernetes workload

### CODE REVIEW

**Comment by stewartshea** [comment]
@jon-funk Just those updates to the readme and then we can push it through. FWIW, I know it's a bit nitpicky, but since those use cases show up on the index, they likely have too much text to fit into a smaller field.

**Comment by jon-funk** [comment]
@stewartshea updated the doc strings

**Review by stewartshea — COMMENTED** [review]
@jon-funk I've directly updated a bunch of the docs. let me know if you're ok with the changes.

File: codebundles/k8s-patroni-lag/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Kubernetes Patroni Replica Lag
+Manage lagging patroni cluster replica members by monitoring their lag and actioning them by reinitializing them with the
+accompanying taskset. Using the Kubernetes API, you can setup the maximum allowed lag of replicas as an SLI and attach an SLO to it
+which triggers an alert when a member passes the allowed lag threshold.
+The taskset will grab a list of replicas who's lag is beyond a configured tolerance and reinitialize them so that they're caught up to the leader.
+
+## Use Cases
+### Use Case: SLI: Measure the maximum lag found across Patroni replicas across a Patroni cluster in Kubernetes
```
[the remark this answers is not in the document]
Comment (jon-funk) [reply codebundles/k8s-patroni-lag/README.md]: Edits look good to me!

**Review by stewartshea — APPROVED** [review]

**Pull request merged**